### ISSUE TITLE
Nodes that execute a function on click should be buttons

### ISSUE THREAD

**Issue by dword-design** [mermaid-js/mermaid#2278]
**Is your feature request related to a problem? Please describe.**
I think clickable nodes that are not linking to a URL but execute a JS function should be `<button>`s for accessibility reasons. Currently they remain divs and cannot be accessed via the tab key.

**Describe the solution you'd like**
If a node has a click event that executes a JS function, it should be rendered as a `<button>`.
